packages/link/src/Link.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+import linkTheme from './linkTheme';
+
+type Props = {
+  /** Content of the links. Can be anything, but see LinkBlock, LinkImage, and LinkTitle. */
+  children: MnrlReactNode,
+  /** URI tied to the Link */
+  href: string
+};
+
+const Root = createStyledComponent(
+  'a',
+  (props, baseTheme) => {
+    const theme = linkTheme(baseTheme);
+
+    return {
+      color: theme.color_link,
+      fontWeight: theme.Link_fontWeight,
+
+      '&:hover': {
+        color: theme.color_link_hover
+      },
+      '&:focus': {
+        color: theme.color_link_focus
+      },
+      // `:active` must be last, to follow LVHFA order:
+      // https://developer.mozilla.org/en-US/docs/Web/CSS/:active
+      '&:active': {
+        color: theme.color_link_active
+      }
+    };
+  },
+  {
+    displayName: 'Link',
+    includeStyleReset: true
+  }
+);
+
+const Content = createStyledComponent('span', (props, baseTheme) => {
+  const theme = linkTheme(baseTheme);
+
+  return {
+    fontSize: theme.Link_fontSize
+  };
+});
+
+/**
+ * Link component
+ */
+export default function Link(props: Props) {
+  const rootProps = {
```

Comment:
You need to pull `children` out so that you're not applying it to `Root`:

```
const rootProps = {
    children,
    ...restProps
};
return <Root {...restProps}>...
```